tests/ha/filesystem.pm

```diff
@@ -28,18 +29,22 @@ sub run {
     my $fs_opts = '-F -N 16';    # Force the filesystem creation and allows 16 nodes
 
     # This Filesystem test can be called multiple time
-    if (read_tag eq 'cluster_md') {
+    if ($tag eq 'cluster_md') {
         $resource = 'cluster_md';
     }
-    elsif (read_tag eq 'drbd_passive') {
+    elsif ($tag eq 'drbd_passive') {
         $resource = 'drbd_passive';
         $fs_lun = '/dev/drbd_passive' if is_node(1);
         $fs_type = 'xfs';
         $fs_opts = '-f';
     }
-    elsif (read_tag eq 'drbd_active') {
+    elsif ($tag eq 'drbd_active') {
         $resource = 'drbd_active';
     }
+    elsif ($tag eq 'skip_fs_test') {
```

Comment:
Isn't this duplicate of line 21 ?
And if it should be at begging because tag drbd_passive would be evaluated before this skip thus not skip.

Reply:
> Isn't this duplicate of line 21 ?

Not exactly the same. L21 would skip the test when _not_ on drbd Maintenance Updates, but continue it if testing a drbd MU, such as in https://openqa.suse.de/tests/9247409#step/drbd_passive/5

L44 on the other hand is skipping the test on scenarios where the test would not work (such as in 3 node) per L47-L51 in `ha/drbd_passive`.

> And if it should be at begging because tag drbd_passive would be evaluated before this skip thus not skip.

Not a problem. $tag would be one or the other, and never both: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/hacluster.pm#L458-L480

(This  probably could've been implemented with `get_var()` and `set_var()` instead, but no idea why Loic went with a local file in the cluster nodes for these tags. Could be a candidate for a re-write in the future)

Reply:
But instead of two places where the test will return there could be just one at the beginning of the if statement. Otherwise fine.

Reply:
Ah, got it. Let me move both exit conditions to the same line and test.

Reply:
New verification runs:

- QAM jobs (3 nodes): [node1](http://mango.qa.suse.de/tests/4886), [node 2](http://mango.qa.suse.de/tests/4887), [node 3](http://mango.qa.suse.de/tests/4888) & [support server](http://mango.qa.suse.de/tests/4885)
- QAM jobs (2 nodes): [node 1](http://mango.qa.suse.de/tests/4882), [node 2](http://mango.qa.suse.de/tests/4883), [client](http://mango.qa.suse.de/tests/4884) & [support server](http://mango.qa.suse.de/tests/4881)
- Alpha Cluster on Milestone Build Validation: [node 1](http://mango.qa.suse.de/tests/4879), [node 2](http://mango.qa.suse.de/tests/4880) & [support server](http://mango.qa.suse.de/tests/4878)

Reply:
Merge this PR, you are writing `skip_fs_test` tag in first commit